src/content/admin/registry-credentials/amazon-ecr.mdx

```diff
@@ -0,0 +1,77 @@
+---
+title: Amazon ECR
+description: Get Private Registry credentials from Amazon ECR 
+sidebar_label: Amazon ECR
+id: amazon-ecr
+---
+
+This tutorial will guide you on how to use a private registry in Amazon Web Services, its necessary credentials, and how to use it in Okteto.
+
+The steps to do it are: 
+
+1. Create User with access to ECR
+1. Get User Credentials
+1. Okteto config with credentials
+1. Add credentials to the Okteto Dashboard
+
+## Step 1: Create User with access to ECR
+
+Create IAM user with `aws cli` ([CLI Reference](https://docs.aws.amazon.com/cli/latest/reference/iam/create-user.html))
```

Comment:
Let's add `aws` as a requirement, as we do in the EKS guide:
https://www.okteto.com/docs/1.19/get-started/install/amazon-eks/#installation-requirements